events/permissions.py

```diff
@@ -27,14 +27,28 @@ def has_permission(self, request, view):
         return request.method in permissions.SAFE_METHODS
 
 
-class UserBelongsToOrganization(BasePermission):
+class UserDataFromRequestMixin:
```

Comment:
A short docstring could be useful for this. I'm sure this is the kind of thing that raises the question "but why?" for future software archaeologists, so an answer for this would be nice.